packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -100,16 +101,20 @@ const ContractDetails = ({
         ''
     );
 
-    const createLookBacksMarker = (abbreviation?: string) => ({
-        label: abbreviation === INDICATIVE_LOW ? localize('Indicative low spot') : localize('Indicative high spot'),
-        icon: (
-            <div className='lookbacks-marker__wrapper'>
-                <Text color='colored-background' size='xxxs' className='lookbacks-marker__asset'>
-                    {abbreviation}
-                </Text>
-            </div>
-        ),
-    });
+    const createLookBacksMarker = (abbreviation?: string) => {
+        const low_spot_text = is_sold ? localize('Low spot') : localize('Indicative low spot');
```

Comment:
Will it be Localize Component rather then localize function?

Reply:
Will apply + expand type, because currently it's accepting only strings.